### PR TITLE
Start on new docs

### PR DESCRIPTION
**Pull request by jimfulton**
Making this a PR because:

- I want to make the request for review explicit.

- My itch to fix docs is so strong, I'm working pretty hard on this. I'm almost finished with a first draft on getting started.

- I see no harm on getting this on master, as the old doc content is preserved (such as it is) in a separate directory.

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+- Software services should include tools for monitoring them.
+  Operations, including monitoring is a software responsibility,
+  because the creators of the software are the ones who know best how
+  to assess whether it is operating correctly.
+
+  It should be possible, when deploying production software, for the
+  software to configure the monitoring system to monitor the software.
+
+- Software should provide facilities to automate it's configuration.
```
Comment (mgedmin) [inline doc/index.rst]: it's -> its

**Comment by jimfulton** [comment]
Semi-rendered here: https://github.com/buildout/buildout/blob/getting-started/doc/index.rst

**Comment by jimfulton** [comment]
@mgedmin have you done more than a cursory review of this?

**Comment by leorochael** [comment]
For more than a decade now, the pypi page for buildout (a.k.a. the `setup.py` `long_description`) has been the de-facto manual for buildout configuration for me.

I suspect this to be true for many buildout users.

If this new manual is to be a replacement for the old `long_description`-as-documentation, I think it would be nice to change `long_description` to have just an intro, and a conspicuous link to a place where this new documentation will be rendered auto-rendered on changes (ReadTheDocs? gh-pages?).

**Comment by jimfulton** [comment]
On Fri, Feb 17, 2017 at 11:51 AM, Leonardo Rochael Almeida <
notifications@github.com> wrote:

> For more than a decade now, the pypi page for buildout (a.k.a. the
> setup.py long_description) has been the de-facto manual for buildout
> configuration for me.
>
> I suspect this to be true for many buildout users.
>
Yup.


> If this new manual is to be a replacement for the old long_description-as-documentation,
> I think it would be nice to change long_description to have just an
> intro, and a conspicuous link to a place where this new documentation will
> be rendered auto-rendered on changes (ReadTheDocs? gh-pages?).
>
Yup. (RTD)

I don't want to do that until the new docs are reasonably complete.

My thought was that, *in the interim*, we'd add a note in the current
long_description pointing people to the new docs, but note that they're not
complete and that the docs below should be used when the new docs are
insufficient.  My thought (subject to change :-)) was that we'd launch the
new topics when topics referenced in getting-started are done, and maybe
when the reference is reasonable populated.  The reference should probably
be relatively easy to throw together based on content we already have.

**Comment by jimfulton** [comment]
Any objections to merging this?

**Comment by mgedmin** [comment]
I'm not sure how non-cursory my review was.  ;)

I don't consider my technical writing skills to be anything to write home about.  I've read the text in `index.rst`.  It made sense to me.

**Comment by jimfulton** [comment]
Thanks for verifying.

BTW, docs committed to master are at http://buildout-docs.readthedocs.io/en/latest/

I'm also trying to make docs.buildout.org a CNAME, but Route 53 isn't  cooperating for some reason.

**Pull request merged**